Source: @pullmoll
Upstream: no
Reason: Musl libc does not have getgrent_r(3)

--- a/src/slurmctld/groups.c	2019-12-20 21:43:45.000000000 +0100
+++ b/src/slurmctld/groups.c	2020-02-23 12:23:19.570618716 +0100
@@ -155,7 +155,7 @@
 	 * databases), the rest of this function essentially does
 	 * nothing.  */
 
-#if defined (__APPLE__)
+#if defined (__APPLE__) || !defined(__GLIBC__)
 	setgrent();
 	while (1) {
 		if ((grp_result = getgrent()) == NULL)
